header.inc

```diff
@@ -224,7 +225,7 @@ function donateWithStripe(subscribe, amount, title, projectId, firstName, lastNa
 </script>
 
 <!-- form validation code -->
-<script src="https://jqueryvalidation.org/files/lib/jquery.js"></script>
```

Comment:
Was this doing anything?  Is the line below a direct replacement for it?  Can it be deleted?

Reply:
yes, we can remove it. done